net/stunnel/Makefile

```diff
@@ -18,7 +18,9 @@ PKG_LICENSE_FILES:=COPYING COPYRIGHT.GPL
 PKG_SOURCE_URL:= \
 	http://ftp.nluug.nl/pub/networking/stunnel/ \
 	http://www.usenix.org.uk/mirrors/stunnel/ \
-	https://www.stunnel.org/downloads/
+	https://www.stunnel.org/downloads/ \
+	https://www.usenix.org.uk/mirrors/stunnel/archive/5.x/
```

Comment:
Thanks for the patch. Better if we transform `PKG_VERSION` to make the url future-proof.  Something like `$(word 1,$(space),$(PKG_VERSION)).x`